plugins/dominant-color-images/readme.txt

```diff
@@ -47,6 +47,12 @@ Contributions are always welcome! Learn more about how to get involved in the [C
 
 == Changelog ==
 
+= 1.1.3 =
```

Comment:
I'm not positive, but I think these will get auto-generated during the deployment process, so no need to include them in the PR. Thanks for being so detailed about this though! ✨

Reply:
That's right. Running `npm run readme` will automatically add the changelog for PRs in the milestone.